### PR TITLE
contribution file and eslint alignment

### PR DESCRIPTION
**Pull request by nadavshlezz**
1. Contribution file created - according to soluto policy in order to align soluto repos properly
2. Eslint alignment - discussed with @TheYarin on it - he will go over my PRs on Eslint in order to set up a baseline alignment as much as I can
  *Added rules
  * set default formatter prettier
  * new extension recommendation
  * changedprettierr config file

### CODE REVIEW

**Review by AleF83 — DISMISSED** [review]

File: .eslintrc [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {
   "root": true,
+  "env": {
+    "browser": true,
```
Comment (tomeresk) [inline .eslintrc:4]: Why would we set `browser` true for a nodejs/backend project?

Comment (nadavshlezz) [reply .eslintrc]: yes - my mistake :) changed it

**Review by tomeresk — DISMISSED** [review]

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,3 +1,4 @@
 {
-    "debug.node.autoAttach": "on"
+  "debug.node.autoAttach": "on",
+  "editor.defaultFormatter": "esbenp.prettier-vscode"
 }
```
Comment (TheYarin) [inline .vscode/settings.json:4]: I would also add the "use `.gitignore`" setting.

File: .eslintrc [hunk 3]
```diff
@@ -1,17 +1,15 @@
 {
   "root": true,
+  "env": {
+    "node": true,
+    "es2021": true
```
Comment (TheYarin) [inline .eslintrc:5]: why not 2022?

**Review by TheYarin — APPROVED** [review]

File: .eslintrc (hunk 3 above)
Comment (nadavshlezz) [reply .eslintrc:5]: image

Comment (nadavshlezz) [reply .eslintrc:5]: according to eslint repo - we cant change it right now - it does not work or will effect - me and yarin have decided to leave it like that so the ecmaVersion should be coordinated with the node/typescript version

**Review by AleF83 — APPROVED** [review]

**Pull request merged**